app/models/banner.rb

```diff
@@ -3,6 +3,10 @@ class Banner < ActiveRecord::Base
   acts_as_paranoid column: :hidden_at
   include ActsAsParanoidAliases
 
+  translates :title,       touch: true
+  translates :description, touch: true
+  globalize_accessors locales: [:en, :es, :fr, :nl, :val, :pt_br]
```

Comment:
When you backport this to CONSUL, create a new commit removing `:val` language key, as Valencian is not available as an "out-of-the-box" language

Worth noting that `pt_br` was giving @voodoorai2000 some problems on #1580 so be careful with it

Reply:
Okay, thanks for the heads up :)